contentcuration/contentcuration/static/js/edit_channel/sharedComponents/Dialog.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <VDialog v-model="dialog" width="400">
+    <VCard>
+      <VCardTitle class="header">
+        <h1>{{ header }}</h1>
+      </VCardTitle>
+
+      <VCardText>
+        <p>{{ text }}</p>
+        <slot name="content"></slot>
+      </VCardText>
+      <VCardActions>
+        <slot name="buttons"></slot>
+      </VCardActions>
+    </VCard>
+  </VDialog>
+</template>
+
+<script>
+
+  export default {
+    name: 'Dialog',
+    $trs: {},
+    props: {
+      header: {
+        type: String,
+        required: true,
+      },
+      text: {
+        type: String,
+        required: true,
+      },
+    },
+    data() {
+      return {
+        dialog: false,
+      };
+    },
+    methods: {
+      prompt() {
```

Comment:
I've included another dialog component in exercise creation PR because of slightly different use case and I planned to merge these two components to one as soon as I know how we're going to use dialog in scope of the whole edit modal so I'd suggest to wait with refactoring after exercise creation review.

Reply:
Sounds good! Seems like waiting for your PR might require the least amount of reworking, so I'll wait for that